docs/src/modules/components/examples/coreExamples.ts

```diff
@@ -101,6 +101,33 @@ const coreExamples = [
     codeSandbox: true,
     stackBlitz: true,
   },
+  {
```

Comment:
I think we should show them at the top of the list, for which you'd need to add them at the beginning just after the functional dashboard (since the component reverses this list when displaying the cards)

Reply:
Ok, just adjusted the order!